File: src/test/java/ac/dnd/server/fixture/TextEncryptorFixture.java
package ac.dnd.server.fixture;

import org.springframework.security.crypto.encrypt.Encryptors;
import org.springframework.security.crypto.encrypt.TextEncryptor;

public class TextEncryptorFixture {
	public static TextEncryptor createNoOpTextEncryptor() {
		return Encryptors.noOpText();
	}
}

File: src/test/kotlin/ac/dnd/server/fixture/TextEncryptorFixture.kt
package ac.dnd.server.fixture

import org.springframework.security.crypto.encrypt.Encryptors
import org.springframework.security.crypto.encrypt.TextEncryptor

object TextEncryptorFixture {
    fun createNoOpTextEncryptor(): TextEncryptor {
        return Encryptors.noOpText()
    }
}